corehq/form_processor/migrations/0092_auto_20200924_1753.py

```diff
@@ -10,12 +13,18 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-        migrations.RemoveIndex(
-            model_name='casetransaction',
-            name='fp_casetrans_formid_05e599_idx',
-        ),
-        migrations.AddIndex(
-            model_name='casetransaction',
-            index=models.Index(fields=['form_id'], name='form_proces_form_id_f2403a_idx'),
-        ),
+        migrations.RunSQL(
+            f"ALTER INDEX {OLD_NAME} RENAME TO {NEW_NAME}",
```

Comment:
whats the reason for doing this rename? @gherceg

Reply:
If you run `makemigrations` on the current code you'll see that Django want's to drop the current index and make a new one with this name.

Reply:
ah, that's weird.